src/main/asciidoc/java/net.adoc

```diff
@@ -797,4 +797,54 @@ NetServerOptions options = new NetServerOptions().
 NetServer server = vertx.createNetServer(options);
 ----
 
-Cipher suite can be specified on the `link:../../apidocs/io/vertx/core/net/NetServerOptions.html[NetServerOptions]` or `link:../../apidocs/io/vertx/core/net/NetClientOptions.html[NetClientOptions]` configuration.
\ No newline at end of file
+Cipher suite can be specified on the `link:../../apidocs/io/vertx/core/net/NetServerOptions.html[NetServerOptions]` or `link:../../apidocs/io/vertx/core/net/NetClientOptions.html[NetClientOptions]` configuration.
+
+==== OpenSSL engine
+
+The default SSL/TLS engine implementation is provided by the JDK.
+
+The engine implementation can be configured to use https://www.openssl.org[OpenSSL] instead. OpenSSL provides
+better performances and CPU usage than the JDK engine, as well as JDK version independence.
+
+OpenSSL requires to configure `link:../../apidocs/io/vertx/core/net/TCPSSLOptions.html#setSslEngine-io.vertx.core.net.SSLEngine-[setSslEngine]` to `link:../../apidocs/io/vertx/core/net/SSLEngine.html#OPENSSL[OPENSSL]`
+and use http://netty.io/wiki/forked-tomcat-native.html[netty-tcnative] jar on the classpath, as well as the correct
+version of OpenSSL installed on your OS.
```

Comment:
this depends if the tcnative jar is statically or dynamically linked, so I would rather remove the part about OpenSSL installation as it is simply wrong

> On Apr 6, 2016, at 10:13 AM, Clement Escoffier <notifications@github.com> wrote:
> 
> In src/main/asciidoc/java/net.adoc <https://github.com/eclipse/vert.x/pull/1353#discussion_r58666838>:
> 
> > \ No newline at end of file
> > +Cipher suite can be specified on the `link:../../apidocs/io/vertx/core/net/NetServerOptions.html[NetServerOptions]` or `link:../../apidocs/io/vertx/core/net/NetClientOptions.html[NetClientOptions]` configuration.
> > +
> > +==== OpenSSL engine
> > +
> > +The default SSL/TLS engine implementation is provided by the JDK.
> > +
> > +The engine implementation can be configured to use https://www.openssl.org[OpenSSL] instead. OpenSSL provides
> > +better performances and CPU usage than the JDK engine, as well as JDK version independence.
> > +
> > +OpenSSL requires to configure `link:../../apidocs/io/vertx/core/net/TCPSSLOptions.html#setSslEngine-io.vertx.core.net.SSLEngine-[setSslEngine]` to `link:../../apidocs/io/vertx/core/net/SSLEngine.html#OPENSSL[OPENSSL]`
> > +and use http://netty.io/wiki/forked-tomcat-native.html[netty-tcnative] jar on the classpath, as well as the correct
> > +version of OpenSSL installed on your OS.
> correct or consistent ?
> 
> Is the tc-native version aligned with the openssl version ?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/eclipse/vert.x/pull/1353/files/c7a6a1ce8a1da75fc0edcccf686f25b82162b149#r58666838>